extensions/google-translate/src/supportedLanguages.json

```diff
@@ -70,8 +70,7 @@
   },
   "Mandarin": {
     "code": "zh-TW",
-    "name": "Taiwanese Mandarin",
-    "flag": "🇹🇼"
```

Comment:
I don't know why the change needs to remove the flag.

Reply:
At the very first place, the flag of this language was written as the flag of PRC, and you changed it to the current flag afterwards. So if this is causing disputations, maybe it's a more neutral and acceptable way to just leave it blank I think. After all, the Traditional Chinese users are more diverse geographically. Leaving it blank doesn't make things worse.